src/tribler-core/tribler_core/modules/libtorrent/download.py

```diff
@@ -402,7 +401,7 @@ def set_selected_files(self, selected_files=None):
     @check_handle(False)
     def move_storage(self, new_dir):
         if not isinstance(self.tdef, TorrentDefNoMetainfo):
-            self.handle.move_storage(new_dir)
+            self.handle.move_storage(str(new_dir))
```

Comment:
Why not cover this line with a test? I think we already have a few mocked `handle` objects, which pattern you can use.

Reply:
Hmm... I would expect it to be covered already.

Reply:
I'll do that in a moment